controllers/openstackdataplanenode_controller.go

```diff
@@ -233,40 +233,46 @@ func (r *OpenStackDataPlaneNodeReconciler) Reconcile(ctx context.Context, req ct
 			return result, nil
 		}
 
-		var novaExternalCompute *novav1beta1.NovaExternalCompute
-		result, novaExternalCompute, err = deployment.DeployNovaExternalCompute(
-			ctx,
-			helper,
-			instance,
-			instance,
-			ansibleSSHPrivateKeySecret,
-			nodeConfigMap,
-			&instance.Status,
-			instance.Spec.NetworkAttachments,
-			instance.Spec.OpenStackAnsibleEERunnerImage)
-		if err != nil {
-			return result, err
-		}
+		novaTemplate := deployment.GetNovaTemplate(instance, instanceRole)
+
+		if novaTemplate != nil {
```

Comment:
I see we do not create the `novaExternalCompute` if the novaTemplate is empty. This is good since we may not alway want the EDPM node to be a compute node.

Reply:
We do need a way to enable/disable it. However, I think we need a common interface for this. What's proposed here wouldn't work for other dataplane services. The ctlplane actually uses an Enabled bool. I'd rather not add a 3rd way.